internal/pb/federation.proto

```diff
@@ -39,16 +39,8 @@ message ContactTracingResponse {
 }
 
 message ContactTracingInfo {
-	// TODO(squee1945 | mikehelmick): this should be an int now.
-	TransmissionRisk transmissionRisk = 1;  // required
+	int32 transmissionRisk = 1;  // required
```

Comment:
Why did you remove the enum? Having it as an enum makes the values self-describing.

Reply:
[starting review]

Reply:
because transmission risk has moved to an integer, between 1-8 where the semantic meaning is not handled by the server